indico/modules/rb_new/client/js/common/bookings/actions.js

```diff
@@ -40,3 +51,25 @@ export function fetchBookingDetails(id) {
 }
 
 export const openBookingDetails = (bookingId) => modalActions.openModal('booking-details', bookingId);
+
+export function changeBookingState(id, action, params = {}) {
+    return async (dispatch) => {
```

Comment:
`return ajaxAction(...);` - no need for the extra wrapper since you do not need to access the state